Parsers/ASimWebSession/Parsers/ASimWebSessionSquidProxy.yaml

```diff
@@ -53,14 +54,28 @@ ParserQuery: |
       UsernameType = "Unknown",
       SrcUsername = iff (SrcUsername == "-", "", SrcUsername), 
       HttpContentType = iff (HttpContentType in (":", "-"), "", HttpContentType), 
-      EventResult = iff (EventResultOriginalDetails has_any ('DENIED', 'INVALID', 'FAIL', 'ABORTED','TIMEOUT') or EventResultDetails >= 400, "Failure", "Success")
-    // -- aliases
+      EventResult = iff (EventOriginalResultDetails has_any ('DENIED', 'INVALID', 'FAIL', 'ABORTED','TIMEOUT') or toint(EventResultDetails) >= 400, "Failure", "Success"),
+      DstIpAddrIsHost = DstIpAddr matches regex "[a-zA-Z]$"
```

Comment:
you are only checking the last character. is that ok?

Reply:
Now that you mentioned that. Not for IPv6 :-). I will think of an alternative.

Reply:
Fixed